tests/test_smoke.py

```diff
@@ -695,6 +741,30 @@ def test_job_queue(generic_cloud: str):
     run_one_test(test)
 
 
+@pytest.mark.lambda_labs
+def test_lambda_job_queue():
+    name = _get_cluster_name()
+    test = Test(
+        'lambda_job_queue',
+        [
+            f'sky launch -y -c {name} {LAMBDA_TYPE} examples/job_queue/cluster.yaml',
+            f'sky exec {name} -n {name}-1 --gpus A100:0.5 -d examples/job_queue/job.yaml',
+            f'sky exec {name} -n {name}-2 --gpus A100:0.5 -d examples/job_queue/job.yaml',
+            f'sky exec {name} -n {name}-3 --gpus A100:0.5 -d examples/job_queue/job.yaml',
+            f'sky queue {name} | grep {name}-1 | grep RUNNING',
+            f'sky queue {name} | grep {name}-2 | grep RUNNING',
+            f'sky queue {name} | grep {name}-3 | grep PENDING',
+            f'sky cancel -y {name} 2',
+            'sleep 5',
+            f'sky queue {name} | grep {name}-3 | grep RUNNING',
+            f'sky cancel -y {name} 3',
```

Comment:
How about keeping these lines the same as `test_job_queue`? We were testing both the job queue for both CPU and GPU requirement.

Reply:
I think we need `--gpus A100:0.5` since we need to override `job.yaml`, which requires `K80:0.5`.